lambdas/build-cri-oauth-request/src/main/java/uk/gov/di/ipv/core/buildcrioauthrequest/helpers/AuthorizationRequestHelper.java

```diff
@@ -129,4 +131,16 @@ public static JWEObject createJweObject(RSAEncrypter rsaEncrypter, SignedJWT sig
             throw new HttpResponseExceptionWithErrorBody(500, ErrorResponse.FAILED_TO_ENCRYPT_JWT);
         }
     }
+
+    private static String getKid(ConfigService configService) {
+        if (ConfigService.isLocal()) {
+            try {
+                return ECKey.parse(configService.getSecret(SIGNING_KEY_JWK)).getKeyID();
```

Comment:
(This means if the signing key is defined with a JWK, it'll use the KID from that - in local it's just `test-signing-key`)